src/System.Runtime/tests/System/GCTests.netstandard1.7.cs

```diff
@@ -67,22 +67,23 @@ public static void GCNotificationNegTests()
             Assert.Throws<ArgumentOutOfRangeException>(() => GC.WaitForFullGCComplete(-2));
         }
 
-        [Fact]
+        [Theory]
+        [InlineData(new object[] { true, -1 })]
```

Comment:
I am not so sure of changing it to theory, as now for each input a new process is going to spun up, i will leave it up to corefx test maintainers on this descision. @stephentoub please reroute\make a call